recipes/libglvnd/meta.yaml

```diff
@@ -30,46 +27,69 @@ requirements:
     - xorg-xextproto
     - xorg-glproto
 
-test:
-  commands:
-    - test -f $PREFIX/lib/libEGL.so.1
-    - test -f $PREFIX/lib/libGL.so.1
-    - test -f $PREFIX/lib/libGLESv1_CM.so.1
-    - test -f $PREFIX/lib/libGLESv2.so.2
-    - test -f $PREFIX/lib/libGLX.so.0
-    - test -f $PREFIX/lib/libOpenGL.so.0
-    - test -f $PREFIX/lib/libGLdispatch.so.0
-    - test -f $PREFIX/lib/pkgconfig/egl.pc
-    - test -f $PREFIX/lib/pkgconfig/gl.pc
-    - test -f $PREFIX/lib/pkgconfig/glesv1_cm.pc
-    - test -f $PREFIX/lib/pkgconfig/glesv2.pc
-    - test -f $PREFIX/lib/pkgconfig/glx.pc
-    - test -f $PREFIX/lib/pkgconfig/opengl.pc
-    - test -f $PREFIX/lib/pkgconfig/libglvnd.pc
-    - test -f $PREFIX/include/EGL/egl.h
-    - test -f $PREFIX/include/EGL/eglext.h
-    - test -f $PREFIX/include/EGL/eglplatform.h
-    - test -f $PREFIX/include/GL/gl.h
-    - test -f $PREFIX/include/GL/glcorearb.h
-    - test -f $PREFIX/include/GL/glext.h
-    - test -f $PREFIX/include/GL/glx.h
-    - test -f $PREFIX/include/GL/glxext.h
-    - test -f $PREFIX/include/GLES/egl.h
-    - test -f $PREFIX/include/GLES/gl.h
-    - test -f $PREFIX/include/GLES/glext.h
-    - test -f $PREFIX/include/GLES/glplatform.h
-    - test -f $PREFIX/include/GLES2/gl2.h
-    - test -f $PREFIX/include/GLES2/gl2ext.h
-    - test -f $PREFIX/include/GLES2/gl2platform.h
-    - test -f $PREFIX/include/GLES3/gl3.h
-    - test -f $PREFIX/include/GLES3/gl31.h
-    - test -f $PREFIX/include/GLES3/gl32.h
-    - test -f $PREFIX/include/GLES3/gl3ext.h
-    - test -f $PREFIX/include/GLES3/gl3platform.h
-    - test -f $PREFIX/include/KHR/khrplatform.h
-    - test -f $PREFIX/include/glvnd/GLdispatchABI.h
-    - test -f $PREFIX/include/glvnd/libeglabi.h
-    - test -f $PREFIX/include/glvnd/libglxabi.h
+outputs:
+  - name: libglvnd
+    files:
+      - **/*.so
+    test:
+      commands:
+        - test -f $PREFIX/lib/libEGL.so.1
+        - test -f $PREFIX/lib/libGL.so.1
+        - test -f $PREFIX/lib/libGLESv1_CM.so.1
+        - test -f $PREFIX/lib/libGLESv2.so.2
+        - test -f $PREFIX/lib/libGLX.so.0
+        - test -f $PREFIX/lib/libOpenGL.so.0
+        - test -f $PREFIX/lib/libGLdispatch.so.0
+  - name: libgl-devel
+    requirements:
+      - {{ pin_subpackage('libglvnd', exact=True) }}
+      - xorg-xextproto
+      - xorg-glproto
+    files:
+      include:
+        - **
+      exclude:
+        - **/*.so
+    test:
+      commands:
+        - test ! -f $PREFIX/lib/libEGL.so.1
+        - test ! -f $PREFIX/lib/libGL.so.1
+        - test ! -f $PREFIX/lib/libGLESv1_CM.so.1
+        - test ! -f $PREFIX/lib/libGLESv2.so.2
+        - test ! -f $PREFIX/lib/libGLX.so.0
+        - test ! -f $PREFIX/lib/libOpenGL.so.0
+        - test ! -f $PREFIX/lib/libGLdispatch.so.0
```

Comment:
these tests will fail. so you need to remove these (as libglvnd will be pulled in)

you can add a a test like:
```
- test ! -f $PREFIX/include/GL/gl.h
```

to the `libglvnd` package instead.

Reply:
Done.